parkhangs-webapp/src/features/modal/ModalParkDetail.js

```diff
@@ -115,6 +135,27 @@ class ModalParkDetail extends React.Component {
                 {this.getSavedParkIcon()}
 
                 <div className="Details">
+                <div className="Section">
+                  <div className="Ratings">
+                    <span className="SectionTitle">Rating</span> <br/>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating(1)}>
+                      <b>1</b>
+                    </button>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating(2)}>
+                      <b>2</b>
+                    </button>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating()}>
+                      <b>3</b>
+                    </button>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating(4)}>
+                      <b>4</b>
+                    </button>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating(5)}>
+                      <b>5</b>
+                    </button>
+                    <span>Average Rating by Users: { this.getAverageRating() }</span>
```

Comment:
have a variable declared in the top of the render function that calls this one so that when the park rating gets updated the average rating also gets updated

Reply:
That might be why some of the comments aren't being updated unless refreshed too, Phil

Reply:
Can't seem to get it to update using this method. Just to make sure, define a const averageRating = this.getAverageRating() in the render method, and then refer to averageRating in my render functionn?

Reply:
@philchapdelaine  yes I think so. See the way I did it with events in the event details modal.